source/blog/2017-05-12-asynchronous-python-sdk.html.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Be asynchronous with Python SDK
+author: omachace
+date: 2017-05-12 08:00:00 UTC
+tags: ovirt, python, sdk, pipelining, async, api
+comments: false
+published: false
+---
+
+Python SDK version 4.1.4 introduced support for sending asynchronous requests and HTTP pipelining. In this blog post we will explain those terms and we will show you an example how to use the Python SDK in asynchronous manner.
+
+## Asynchronous requests
```

Comment:
What are async requests? Why are they good for? At least refer to some external explanation.